engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/partials/sync-status.html

```diff
@@ -14,6 +14,11 @@
     Weekly on {{ product.sync_plan.sync_date | date:'EEEE' }} at {{ product.sync_plan.sync_date | date:'mediumTime' }} (Server Time)
   </span>
 </dd>
+<dd ng-if="product.sync_plan.interval == 'custom cron'">
```

Comment:
would `product.sync_plan.cron_expression` only exist if its  has a 'custom cron' interval? If so, could we use that to check?

Reply:
Yes..that's right..For a sync plan with custom cron interval we need to have cron expression and vice versa..So both are the same check..

Reply:
I'm thinking you could just do:
```html
<dd ng-if="product.sync_plan.cron_expression'">
```
Since that is what is actually displaying?

Reply:
Updated..